lib/app/app.dart

```diff
@@ -1,13 +1,24 @@
 import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:todo_api/todo_api.dart';
 import 'package:todo_bloc/home/home.dart';
+import 'package:todo_repository/todo_repository.dart';
 
 class App extends StatelessWidget {
-  const App({Key? key}) : super(key: key);
+  final TodoApi todoApi;
```

Comment:
My advice: do not inject API/data layers in the app. Instead, you can do this in `main`/`bootstrap` files, and only inject repositories/business rules ;)

That way, also you avoid making the app responsible for creating Repositories, like we see in line 18 ;)

Reply:
Now i got the main idea to use `RepositoryProvider.value` instead of `RepositoryProvider`

Thanks Jorge!